src/components/Faq/Faq.js

```diff
@@ -100,12 +125,32 @@ const Faq = ({ currentItemData, navData, faqHtml, crumbs, lang }) => {
     };
   }, []);
 
-  React.useEffect(() => {
+  useEffect(() => {
     return () => {
       document.body.removeAttribute(`data-filters`);
     };
   }, []);
 
+  const removeScript = (script) => {
```

Comment:
This gets redefined on every render. Since the function is not reused anywhere else, could it's body be moved inside the `useEffect`?

Something like
```
return () => {
  if (script) document.body.removeChild(script);
};
```